llvm/include/llvm/TargetParser/AArch64TargetParser.h

```diff
@@ -173,6 +173,8 @@ enum ArchExtKind : unsigned {
   AEK_SMEF8F16 =      69, // FEAT_SME_F8F16
   AEK_SMEF8F32 =      70, // FEAT_SME_F8F32
   AEK_SMEFA64 =       71, // FEAT_SME_FA64
+  AEK_JSCVT =         72, // FEAT_JSCVT
+  AEK_FCMA =          73, // FEAT_FCMA
```

Comment:
Hi,
As you are updating this enum, I think you need to update the related tests also in the TargetParserTest.cpp, the test is named: `AArch64ExtensionFeatures`

Reply:
Good points. The tests have been added.